pom.xml

```diff
@@ -93,18 +93,19 @@
       <optional>true</optional>
     </dependency>
     <dependency>
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-context-support</artifactId>
```

Comment:
Why spring-context-support instead of just spring-context?

Reply:
Oh... sorry, I misunderstood that the `MessageSourceAccessor` is in spring-context-support.

Reply:
Fix via 74ad91d17366ec342291339b28d2f59a85c64bd1.